joomla/joomla-cms
# [5.4] Added delay to form validation blur event

Pull Request for Issue #39419, #32692.

### Summary of Changes

Added a tiny delay to the js form validation blur event, to prevent execution before the click event is fired on other fields/buttons.

### Testing Instructions

1. Create a Users > Login Form menu item
2. Go to the menu item
3. Set the focus to the Username field (should be autofocused) and ensure the field is blank
4. Tick the 'Remember me' checkbox

### Actual result BEFORE applying this Pull Request

User would need tick the checkbox twice

### Expected result AFTER applying this Pull Request

Tick box functions as expected

### Link to documentations
Please select:
